pages/styles.css

```diff
@@ -0,0 +1,7 @@
+li {
+    list-style-type: none;
+}
+
+a {
```

Comment:
Please use the next.js ``Link`` Component

Reply:
Hi @Roland-Hufnagel,

Thanks for pointing that out! I thought the Next.js Link component was only for internal routes. However, I'm facing challenges when trying to style the Link component alongside styled React components. A standard anchor tag seems simpler to design, doesn't it?

Reply:
I'm using the Next.js Link component. However, I'm using a styled a tag within the Link component for styling purposes, as I've faced challenges directly styling the Link component itself.